build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
   id 'jacoco'
   id "org.flywaydb.flyway" version "7.0.2"
   id 'io.spring.dependency-management' version '1.1.0'
-  id 'org.springframework.boot' version '2.7.12'
+  id 'org.springframework.boot' version '3.1.0'
```

Comment:
spring 3.1.0 is not compatible with flyway 9.0 according to https://github.com/spring-projects/spring-boot/issues/35721, spring 3.1.1 with fix is available

Reply:
changed to 3.1.1